nipype/interfaces/freesurfer/longitudinal.py

```diff
@@ -0,0 +1,184 @@
+                x) for x in self.inputs.transform_outputs]
+        if isdefined(self.inputs.scaled_intensity_outputs):
+            outputs['scaled_intensity_outputs'] = [os.path.abspath(
+                x) for x in self.inputs.scaled_intensity_outputs]
+        return outputs
+
+
+class FuseSegmentationsInputSpec(FSTraitedSpec):
+    # required
+    subject_id = traits.String(argstr='%s', position=-3,
+                               desc="subject_id being processed")
+    timepoints = InputMultiPath(
+        traits.String(), mandatory=True, argstr='%s', position=-2,
+        desc='subject_ids or timepoints to be processed')
+    out_file = File(exists=False, mandatory=True, position=-1,
+                    desc="output fused segmentation file")
+    in_segmentations = InputMultiPath(
+        File(exists=True), argstr="-a %s", mandatory=True,
+        desc="name of aseg file to use (default: aseg.mgz) \
+        must include the aseg files for all the given timepoints")
+    in_segmentations_noCC = InputMultiPath(
+        File(exists=True), argstr="-c %s", mandatory=True,
+        desc="name of aseg file w/o CC labels (default: aseg.auto_noCCseg.mgz) \
+        must include the corresponding file for all the given timepoints")
+    in_norms = InputMultiPath(
+        File(exists=True), argstr="-n %s", mandatory=True,
+        desc="-n <filename>  - name of norm file to use (default: norm.mgs) \
+        must include the corresponding norm file for all given timepoints \
+        as well as for the current subject")
+
+class FuseSegmentationsOutputSpec(TraitedSpec):
+    out_file = File(exists=False, desc="output fused segmentation file")
+
+class FuseSegmentations(FSCommand):
+
+    """ fuse segmentations together from multiple timepoints
+
+    Examples
+    --------
+    >>> from nipype.interfaces.freesurfer import FuseSegmentations
+    >>> fuse = FuseSegmentations()
+    >>> fuse.inputs.subject_id = 'tp.long.A.template'
+    >>> fuse.inputs.timepoints = ['tp1', 'tp2']
+    >>> fuse.inputs.out_file = 'tp.long.A.template/mri/aseg.fused.mgz' #doctest: +SKIP
+    >>> fuse.inputs.in_segmentations = ['tp1/mri/aseg.presurf.mgz', 'tp1/mri/aseg.presurf.mgz'] #doctest: +SKIP
+    >>> fuse.inputs.in_segmentations_noCC = ['tp1/mri/aseg.auto_noCCseg.mgz', 'tp1/mri/aseg.auto_noCCseg.mgz'] #doctest: +SKIP
+    >>> fuse.inputs.in_norms = ['tp1/mri/norm.mgz', 'tp1/mri/norm.mgz', 'tp.long.A.template/mri/norm.mgz'] #doctest: +SKIP
+    >>> fuse.cmdline #doctest: +SKIP
+    """
+
+    _cmd = 'mri_fuse_segmentations'
+    input_spec = FuseSegmentationsInputSpec
+    output_spec = FuseSegmentationsOutputSpec
+
+    def _format_arg(self, name, spec, value):
+        if name in ('in_segmentations', 'in_segmentations_noCC', 'in_norms'):
+            # return enumeration value
+            return spec.argstr % os.path.basename(value[0])
+        return super(FuseSegmentations, self)._format_arg(name, spec, value)
+
+    def _list_outputs(self):
```

Comment:
@satra  I'm not quite sure of what you are asking for here.  Is there another nipype tools that we could use as a reference that uses "name source" and "name template" paradigm?